aws/scenarios/vm/os/windows.go

```diff
@@ -1,29 +1,34 @@
 package os
 
 import (
+	"fmt"
+
 	"github.com/DataDog/test-infra-definitions/aws"
 	"github.com/DataDog/test-infra-definitions/aws/ec2/ec2"
-	oscommon "github.com/DataDog/test-infra-definitions/common/os"
+	commonos "github.com/DataDog/test-infra-definitions/common/os"
 )
 
 type windows struct {
-	*oscommon.Windows
+	*commonos.Windows
 	env aws.Environment
 }
 
 func newWindows(env aws.Environment) *windows {
 	return &windows{
-		Windows: oscommon.NewWindows(&env),
+		Windows: commonos.NewWindows(&env),
 		env:     env,
 	}
 }
 
-func (*windows) GetSSHUser() string { panic("Not Yet supported") }
+func (*windows) GetSSHUser() string { return "administrator" }
```

Comment:
There's a diff with the RDP user which is `Administrator`, is it expected?

Reply:
Both work. Updated to `Administrator`.